app/src/main/java/erevacation/com/userlist/ui/homescreen/profile/ProfileFragment.kt

```diff
@@ -0,0 +1,21 @@
+package erevacation.com.userlist.ui.homescreen.profile
+
+import android.os.Bundle
+import erevacation.com.userlist.R
+import erevacation.com.userlist.basic.BasicFragment
+import erevacation.com.userlist.databinding.FragmentProfileBinding
+
+class ProfileFragment : BasicFragment<ProfileContract.ProfilePresenter, FragmentProfileBinding>(), ProfileContract.ProfileView {
```

Comment:
This line is tooo long. Can we move `: BasicFragment<ProfileContract.ProfilePresenter, FragmentProfileBinding>(), ProfileContract.ProfileView {` in second line?